sdk/search/search-documents/review/search-documents.api.md

```diff
@@ -1596,7 +1600,7 @@ export interface SearchIndex {
 
 // @public
 export class SearchIndexClient {
-    constructor(endpoint: string, credential: KeyCredential, options?: SearchIndexClientOptions);
+    constructor(endpoint: string, credential: KeyCredential, apiVersion?: string, options?: SearchIndexClientOptions);
```

Comment:
I don't like having multiple optional parameters. This forces anyone who wants to pass `options` to pass `undefined` to `apiVersion` if they don't want to set it, which feels bad.

Why can't `apiVersion` live on `SearchIndexClientOptions` ?

Reply:
Agreed. I will change it for all the 3 clients in the next commit.

Reply:
@xirzec Updated in the latest commit